test/components/translation_nav_test.rb

```diff
@@ -0,0 +1,82 @@
+require 'component_test_helper'
+
+class TranslationNavTest < ComponentTestCase
+  def component_name
+    "translation-nav"
+  end
+
+  test "fails to render a translation nav when no translations are given" do
+    assert_raise do
+      render_component({})
+    end
+  end
+
+  test "renders an active translation nav item with a text description" do
+    render_component(
+      translations: [
+        {
+          locale: 'en',
+          base_path: '/en',
+          text: 'English',
+          active: true
+        },
+        {
+          locale: 'hi',
+          base_path: '/hi',
+          text: 'हिंदी',
+        }
+      ]
+    )
+
+    assert_select ".app-c-translation-nav__list__language", text: "English"
+  end
+
+  test "does not render an active translation item with a link" do
+    render_component(
+      translations: [
+        {
+          locale: 'en',
+          base_path: '/en',
+          text: 'English',
+          active: true
+        },
+        {
+          locale: 'hi',
+          base_path: '/hi',
+          text: 'हिंदी',
+        }
+      ]
+    )
+
+    assert_select ".app-c-translation-nav__list__language a[href=\"/en\"]",
+                    false, "An active item should not link to a page translation"
```

Comment:
We should try to do positive assertions, rather than asserting something does not exist since this test does not confirm that there is an active element formatted how we expect, just that it's not a link. For example it could not be there at all and the tests would pass.